rfc-018-our-decision-making-process-is-public.md

```diff
@@ -0,0 +1,72 @@
+# Our decision making process is public
+
+## Summary
+
+Our process of making decisions and the decisions themselves may be read by
+all staff and members of the public.
+
+This topic has previously been [debated as an issue](https://github.com/dxw/tech-team-rfcs/issues/2)
+on this repository. The decision to take no action should still be persisted
+as a formal decision so has become an RFC.
+
+## Problem
+
+This repository started as public by default but do we agree that this is right
+in this case?
+
+Reasons for:
+
+- dxw values include being honest and we agree with the
+  [principle of making things open as it makes things better](https://www.gov.uk/guidance/government-design-principles#make-things-open-it-makes-things-better)
+- [the dxw playbook is public](https://github.com/dxw/playbook) and has not
+ caused us concern or prevented us from iterating it
+- others can learn from us and spend more their time making things better for
+  users
+- it allows us to share our thinking and rationale on our technical decisions,
+  doing so publicly as well as privately can increase trust and empathy with
+  more people
+- with the potential for the world watching, the bar is raised.
+  being open from the start ensures that there is less risk to us should a
```

Comment:
Should this be a new bullet?

Reply:
I _think_ it's a continuation of the previous point. Probably could do with a captial letter.